src/createServer.js

```diff
@@ -1,11 +1,17 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const expensesRoutes = require('./routes/expenses.js');
+const usersRoutes = require('./routes/users.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
```

Comment:
clear `users` and `expenses` on each server creation (just reassign empty array)